interface/main/calendar/add_edit_event.php

```diff
@@ -1091,27 +1096,37 @@ function set_category() {
 // radio buttons are clicked.
 function set_allday() {
     var f = document.forms[0];
+    var s = f.form_category;
     var color1 = 'var(--gray)';
     var color2 = 'var(--gray)';
-    var disabled2 = true;
+    var timeDisabled = true;
```

Comment:
using **var** is consistent with the older code, in the future I'd recommend changing all of these local variables to use **let** or **const** instead of **var** if you're going to heavily modify a function. I had to pull down the code and make sure there wasn't a ```var disabled``` already in the file since the variable was called **disabled2**.  Using **let** would have avoided that problem as it would have kept the variable local to the function.